src/twin/mod.rs

```diff
@@ -336,6 +336,8 @@ impl Twin {
 
         let mut signals = Signals::new(TERM_SIGNALS)?;
 
+        systemd::sd_notify_ready();
```

Comment:
we initialize the watchdog afterwarts, if that fails we have a theoretical problem with the systemd state machine.
e.g. if units depends on ods these would start, even if ods is not functioning.